sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/models/_conversation_analysis_client_enums.py

```diff
@@ -11,6 +11,56 @@
 from azure.core import CaseInsensitiveEnumMeta
 
 
+class AgeUnit(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
+    """The Age Unit of measurement
+    """
+
+    UNSPECIFIED = "Unspecified"
+    YEAR = "Year"
+    MONTH = "Month"
+    WEEK = "Week"
+    DAY = "Day"
+
+class AnalyzeConversationTaskKind(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
```

Comment:
We don't usually have enums with a single value.
Are customers expected to set this on input?
If so - is it expected to expand to more options in future?

Reply:
Yes. in an effort to unify with TA in the upcoming releases they decided to name this for now "CustomConversation"